moped-database/views/component_arcgis_online_view.sql

```diff
@@ -13,7 +13,8 @@ CREATE OR REPLACE VIEW component_arcgis_online_view AS WITH work_types AS (
 council_districts AS (
     SELECT
         features.component_id AS project_component_id,
-        string_agg(DISTINCT features_council_districts.council_district_id::text, ', '::text) AS council_districts
+        string_agg(DISTINCT features_council_districts.council_district_id::text, ', '::text) AS council_districts,
+        string_agg(DISTINCT lpad(features_council_districts.council_district_id::text, 2, '0'::text), ', '::text) AS council_districts_searchable
```

Comment:
TIL. postgres rules!

Reply:
yeah - `lpad` was new to me. they've thought of everything!